downstream/modules/platform/con-storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -1,12 +1,15 @@
 [id="con-storage-options-for-operator-installation-on-ocp_{context}"]
 = Storage options for {OperatorPlatform} installation on {OCP}
 
-If you are using file-based storage and your installation scenario includes {HubName}, ensure that you change the `ReadWriteOnce` default storage option for {OperatorPlatform} to `ReadWriteMany`.
-
 {HubNameStart} requires `ReadWriteMany` file-based storage, Azure Blob storage, or Amazon S3-compliant storage for operation so that multiple pods can access shared content, such as collections.
 
+The process for configuring object storage on the `AutomationHub` CR is similar for Amazon S3 and Azure Blob Storage.
+
+If you are using file-based storage and your installation scenario includes {HubName}, ensure that you change the `ReadWriteOnce` default storage option for {OperatorPlatform} to `ReadWriteMany`.
```

Comment:
The default is ReadWriteMany when using file-based and you can't actually select ReadWriteOnce in the UI form (you would have to manually specify it in the yaml view.  

So we might want to re-word this a bit.

Reply:
@rooftopcellist Would the following work?

If you are using file-based storage and your installation scenario includes {HubName}, ensure that the storage option for {OperatorPlatform} is set to `ReadWriteMany`.
`ReadWriteMany` is the default storage option.